src/jaxsim/parsers/rod/parser.py

```diff
@@ -113,6 +114,23 @@ def extract_model_data(
     # Create a dictionary to find easily links
     links_dict: Dict[str, descriptions.LinkDescription] = {l.name: l for l in links}
 
+    # ============
+    # Parse frames
+    # ============
+
+    # Parse the frames (unconnected)
+    frames = [
+        descriptions.LinkDescription(
+            name=f.name,
+            mass=jnp.array(0.0, dtype=float),
+            inertia=jnp.zeros(shape=(3, 3)),
+            parent=links_dict[f.attached_to],
+            pose=f.pose.transform() if f.pose is not None else jnp.eye(4),
```

Comment:
In which case it make sense for `f.pose` to be `None` ? cc @xela-95 @diegoferigo

Reply:
I guess if the frame is overlapping with its parent, specified in the `attached_to` attribute.

Reply:
Refer to http://sdformat.org/tutorials?tut=pose_frame_semantics&cat=specification&#frame for further details.